contents/tutorials/hogql-date-time-filters.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using HogQL for advanced time and date filters
+date: 2023-05-19
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-9.png
+tags: ["hogql", "insights"]
+---
+
+Because there are infinite ways to break down time, there are infinite ways to filter based on time. HogQL unlocks more of these in PostHog, and in this tutorial, we go through examples of how to use do that.
+
+To add a HogQL filter, create a new insight, open the filter dropdown, click "Add filter" below your data series, select HogQL from the options, write your expression, and click "Add HogQL expression." HogQL filters are available on every type of insight from trends to funnels to lifecycle.
+
+![hogql.mp4](../images/tutorials/hogql-date-time-filters/hogql.mp4)
+
+## Accessing your data’s dates and times
+
+To filter on dates and times, you need access to those values. You can find a full list of events, properties, and types in your [data management tab](https://app.posthog.com/data-management/events), but here is a summary of the important points:
+
+- You can access the time events happened with the `timestamp` property which is automatically set on events.
+- Persons have a `created_at` property which you can access with `person.created_at`.
```

Comment:
When referring to posthog concept, like persons, for the first time, we should hyper link to it

```suggestion
- [Persons](/docs/data/persons) have a `created_at` property which you can access with `person.created_at`.
```